### PR TITLE
German README.md

### PR DESCRIPTION
**Pull request by FierceSkit**
Added a German README.md, since there'd been important changes lately.
Please Read & Review - any Suggestions, aswell as Corrections welcome.

### CODE REVIEW

**Comment by patricklodder** [comment]
Added `help wanted` -> needs a native speaker to review.

File: doc/intl/de_GER/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+ 
+## Verwendung 💻
+ 
+Um Dogecoin Core zu verwenden, sehen Sie sich Bitte die [Installations-Anleitung](INSTALL.md) und das [Einstiegstutorial](doc/getting-started.md) an.
+ 
+Die in Dogecoin Core enthaltene JSON-RPC-API ist selbstdokumentiert und kann mit dem Befehl `dogecoin-cli help` eingesehen werden. Detailliertere Informationen zu jedem Befehl finden Sie unter `dogecoin-cli help <command>`. Alternativ können Sie sich die [Bitcoin Core-Dokumentation](https://developer.bitcoin.org/reference/rpc/) ansehen, die ein ähnliches Protokoll implementiert hat, um eine navigierbare Version zu erhalten.
+ 
+### Viele Ports
+
+Dogecoin Core verwendet den Port `22556` als Standart-Port zur Kommunkation
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:35]:
```suggestion
Dogecoin Core verwendet den Port 22556 als Standardport zur Kommunikation
```

Changed "Standart-Port" to "Standardport" and "Kommunkation" to "Kommunikation" for correct spelling.

File: doc/intl/de_GER/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+mit dem Peer-to-Peer Netzwerk und um die "Mainnet" Blockchain zu Synchronisieren.
+Dies ist notwendig um über neue Transaktionen und Blöcke informiert zu bleiben.
+Zusätzlich kann ein JSONRPC Port geöffnet werden, welcher den Port `22555` als Standartport für Mainnet Nodes verwendet.
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:38]:
```suggestion
Zusätzlich kann ein JSON-RPC-Port geöffnet werden, der den Port 22555 als Standardport für Mainnet-Nodes verwendet.
```

Added hyphens for "JSON-RPC-Port" and "Mainnet-Nodes" to match German compound noun formatting.

File: doc/intl/de_GER/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+**Es wird STRENGSTENS davon abgeraten RPC Ports im öffentlichen Internet Sichtbar zu machen!**
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:40]:
```suggestion
**Es wird dringend davon abgeraten, RPC-Ports im öffentlichen Internet sichtbar zu machen!**
```

Changed "STRENGSTENS" to "dringend" for a more typical expression and corrected "Sichtbar" to "sichtbar."

File: doc/intl/de_GER/README.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+| Funktion | mainnet | testnet | regtest |
+| :------- | ------: | ------: | ------: |
+| P2P      |   22556 |   44556 |   18444 |
+| RPC      |   22555 |   44555 |   18332 |
+
+## Laufende Entwicklungen - Fahrplan zum Mond 🌒
+
+Dogecoin Core ist eine Open-Source Software und wird von der Community entwickelt.
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:49]:
```suggestion
Dogecoin Core ist eine Open-Source-Software und wird von der Community entwickelt.
```

Added a hyphen to "Open-Source-Software" for correct compound noun formatting.

File: doc/intl/de_GER/README.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+Der Entwicklungsprozess ist transparent und öffentlich Einsehbar; Jeder kann ihn sehen, darüber diskutieren und daran teilhaben!
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:50]:
```suggestion
Der Entwicklungsprozess ist transparent und öffentlich einsehbar; jeder kann ihn sehen, darüber diskutieren und daran teilhaben!
```

Lowercased "Einsehbar" to "einsehbar" and "Jeder" to "jeder" for consistency in capitalization.

File: doc/intl/de_GER/README.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+* [GitHub Projekte](https://github.com/dogecoin/dogecoin/projects) 
+  Wird verwendet, um den bereits geplanten und derzeit in Entwicklung befindlichen Releases zu folgen.
+* [GitHub Diskussionen](https://github.com/dogecoin/dogecoin/discussions)
+  Wird verwendet, um geplante und ungeplante Funktionen (Features) zu diskutieren, die sich auf die Entwicklung der Dogecoin Core-Software, die zugrunde liegenden Protokolle und das DOGE-Asset beziehen.
+* [Dogecoindev subreddit](https://www.reddit.com/r/dogecoindev/)
+
+### Versionsstrategie
+
+Die Vergabe der Versionsnummern efolgt nach dem Prinzip von ```major.minor.patch```.
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:62]:
```suggestion
Die Vergabe der Versionsnummern erfolgt nach dem Prinzip von ```major.minor.patch```.
```

Corrected "efolgt" to "erfolgt".

File: doc/intl/de_GER/README.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+### Branches
+Es gibt 4 Arten von "Branches" in diesem Repository:
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:65]:
```suggestion
Es gibt 4 Arten von Branches in diesem Repository:
```

Removed quotation marks around "Branches" for consistency.

File: doc/intl/de_GER/README.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+***Reichen Sie Ihre Pull-Requests im `master`-Branch ein!***
+
+* Die Wartungs-Branches sind ausschließlich durch Freigabe veränderbar. Wenn eine Veröffentlichung geplant ist, wird ein Entwicklungs-Branch erstellt und die Beitrage ("Commits") werden von den Betreuern aus dem Master-Branch ausgewählt.
+
+## Beitragen 🤝
+ 
+Falls Sie einen Fehler oder Probleme beim Verwenden dieser Software finden, melden Sie diese über das vorhandene [Ticket System](https://github.com/dogecoin/dogecoin/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5Bbug%5D+).
+ 
+Wenn Sie an der Mithilfe oder der Entwicklung von Dogecoin Core interessiert sind sehen Sie sich Bitte den [Beitragsleitfaden](CONTRIBUTING.md) an.
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:80]:
```suggestion
Wenn Sie an der Mithilfe oder der Entwicklung von Dogecoin Core interessiert sind, sehen Sie sich bitte den [Beitragsleitfaden](CONTRIBUTING.md) an.
```

Added a comma and lowercased "Bitte" for correct punctuation and consistency.

File: doc/intl/de_GER/README.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+Oft gibt es Themen, die [Hilfe](https://github.com/dogecoin/dogecoin/labels/help%20wanted) benötigen. Ihr Beitrag könnte einen hohen Stellenwert haben und wird daher auch sehr geschätzt.
+ 
+## Communities 🚀🍾
+ 
+Sie können den Communities auf verschiedenen sozialen Medien beitreten.
+Herzlich sind sie Eingeladen sich umzusehen um neue Leute kennenzulernen, die neueste Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
+Vielleicht findet sich ja auch Jemand, der Ihnen bei ihren Projekten behilflich sein kann.
```
Comment (loom4k) [inline doc/intl/de_GER/README.md:87]:
```suggestion
Vielleicht findet sich ja auch jemand, der Ihnen bei Ihren Projekten behilflich sein kann.
```

Lowercased "Jemand" and changed "ihren" to "Ihren" to match the capitalization and case correctly.

**Review by loom4k — CHANGES_REQUESTED** [review]
I would change a couple of things, mainly spelling mistakes and incorrect capitalization. Overall a really good translation.

File: doc/intl/de_GER/README.md (hunk 7 above)
Comment (loom4k) [reply doc/intl/de_GER/README.md:65]: Sounds good to me. Let's keep it as you had it.

File: doc/intl/de_GER/README.md (hunk 1 above)
Comment (loom4k) [reply doc/intl/de_GER/README.md:35]: I made a typo in my suggestion and forgot to add the back braces. That's on me. Definitely keep it as `22556`.

File: doc/intl/de_GER/README.md (hunk 2 above)
Comment (loom4k) [reply doc/intl/de_GER/README.md:38]: Again, typo on my side for the back quotation marks.

File: doc/intl/de_GER/README.md (hunk 3 above)
Comment (loom4k) [reply doc/intl/de_GER/README.md:40]: No problem with that. Let's keep it as you had it 👍

**Comment by loom4k** [comment]
Looks all good to me.

**Review by loom4k — APPROVED** [review]
Looks all good to me.

**Comment by loom4k** [comment]
Absolutely. I will do this as soon as I get home. Might be a couple of hours however. Will leave a CR ASAP.

**Review by loom4k — APPROVED** [review]
This looks all good on my side. Great translation work by @FierceSkit! @patricklodder I'd say that's a wrap. This would be ready to merge when you are 😉

**Comment by loom4k** [comment]
No problem. I didn't think of leaving it open but it's actually a great idea. I'm so used of being in a rush. Let's see what others think!

File: doc/intl/de_GER/README.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+ 
+Falls Sie einen Fehler oder Probleme beim Verwenden dieser Software finden, melden Sie diese über das vorhandene [Ticket System](https://github.com/dogecoin/dogecoin/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5Bbug%5D+).
+ 
+Wenn Sie an der Mithilfe oder der Entwicklung von Dogecoin Core interessiert sind, sehen Sie sich bitte den [Beitragsleitfaden](CONTRIBUTING.md) an.
+Oft gibt es Themen, die [Hilfe](https://github.com/dogecoin/dogecoin/labels/help%20wanted) benötigen. Ihr Beitrag könnte einen hohen Stellenwert haben und wird daher auch sehr geschätzt.
+ 
+## Communities 🚀🍾
+ 
+Sie können den Communities auf verschiedenen sozialen Medien beitreten.
+Herzlich sind sie Eingeladen sich umzusehen um neue Leute kennenzulernen, die neueste Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
```
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:86]:
```suggestion
Herzlich sind sie Eingeladen sich umzusehen um neue Leute kennenzulernen, die neuesten Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
```
Changed singular to plural.

File: doc/intl/de_GER/README.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+ 
+Falls Sie einen Fehler oder Probleme beim Verwenden dieser Software finden, melden Sie diese über das vorhandene [Ticket System](https://github.com/dogecoin/dogecoin/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5Bbug%5D+).
+ 
+Wenn Sie an der Mithilfe oder der Entwicklung von Dogecoin Core interessiert sind, sehen Sie sich bitte den [Beitragsleitfaden](CONTRIBUTING.md) an.
+Oft gibt es Themen, die [Hilfe](https://github.com/dogecoin/dogecoin/labels/help%20wanted) benötigen. Ihr Beitrag könnte einen hohen Stellenwert haben und wird daher auch sehr geschätzt.
+ 
+## Communities 🚀🍾
+ 
+Sie können den Communities auf verschiedenen sozialen Medien beitreten.
+Herzlich sind sie Eingeladen sich umzusehen um neue Leute kennenzulernen, die neuesten Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
```
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:86]:
```suggestion
Herzlich sind Sie eingeladen sich umzusehen um neue Leute kennenzulernen, die neuesten Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
```
Spelling.

**Review by DGalapagos8 — APPROVED** [review]
looks good to me.

File: doc/intl/de_GER/README.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+### Versionsstrategie
+
+Die Vergabe der Versionsnummern erfolgt nach dem Prinzip von ```major.minor.patch```.
+
+### Branches
+Es gibt 4 Arten von sog. "Branches" (Unterverzweigungen) in diesem Repository:
+
+- **master:** Instabil - Enthält den neuesten Code, der sich derzeit in der Entwicklung befindet.
+- **maintenance:** Stabil, enthält die neueste Version früherer Versionen, die noch aktiv gewartet werden. Format: ```<version>-maint```
+- **development:** Intabil, enthält neuen Code für kommende Versionen. Format: ```<version>-dev```
```
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:69]:
```suggestion
- **development:** Instabil, enthält neuen Code für kommende Versionen. Format: ```<version>-dev```
```

**Review by DGalapagos8 — CHANGES_REQUESTED** [review]

File: doc/intl/de_GER/README.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+## Verwendung 💻
+ 
+Um Dogecoin Core zu verwenden, sehen Sie sich Bitte die [Installations-Anleitung](INSTALL.md) und das [Einstiegstutorial](doc/getting-started.md) an.
+ 
+Die in Dogecoin Core enthaltene JSON-RPC-API ist selbstdokumentiert und kann mit dem Befehl `dogecoin-cli help` eingesehen werden. Detailliertere Informationen zu jedem Befehl finden Sie unter `dogecoin-cli help <command>`. Alternativ können Sie sich die [Bitcoin Core-Dokumentation](https://developer.bitcoin.org/reference/rpc/) ansehen, die ein ähnliches Protokoll implementiert hat, um eine navigierbare Version zu erhalten.
+ 
+### Viele Ports
+
+Dogecoin Core verwendet den Port `22556` als Standart-Port zur Kommunkation
+mit dem Peer-to-Peer Netzwerk und um die "Mainnet" Blockchain zu Synchronisieren.
```
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:36]:
```suggestion
mit dem Peer-to-Peer Netzwerk und um die "Mainnet" Blockchain zu synchronisieren.
```

File: doc/intl/de_GER/README.md [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+ 
+## Verwendung 💻
+ 
+Um Dogecoin Core zu verwenden, sehen Sie sich Bitte die [Installations-Anleitung](INSTALL.md) und das [Einstiegstutorial](doc/getting-started.md) an.
+ 
+Die in Dogecoin Core enthaltene JSON-RPC-API ist selbstdokumentiert und kann mit dem Befehl `dogecoin-cli help` eingesehen werden. Detailliertere Informationen zu jedem Befehl finden Sie unter `dogecoin-cli help <command>`. Alternativ können Sie sich die [Bitcoin Core-Dokumentation](https://developer.bitcoin.org/reference/rpc/) ansehen, die ein ähnliches Protokoll implementiert hat, um eine navigierbare Version zu erhalten.
+ 
+### Viele Ports
+
+Dogecoin Core verwendet den Port `22556` als Standart-Port zur Kommunkation
```
Comment (DGalapagos8) [inline doc/intl/de_GER/README.md:35]:
```suggestion
Dogecoin Core verwendet den Port `22556` als Standart-Port zur Kommunikation
```

**Review by DGalapagos8 — CHANGES_REQUESTED** [review]
Those are the last things I found. spelling and grammer.

**Review by DGalapagos8 — APPROVED** [review]
I totally approve this.

**Comment by patricklodder** [comment]
I'll squash this for you.

**Review by patricklodder — APPROVED** [review]
utACK. This still needs integration with the index in `doc/intl/README.md`, but that could be done separately too.

Since I squashed this, needs a second ACK from another maintainer.

**Review by michilumin — APPROVED** [review]
Drawing on my German grom growing up in a weird school where it was spoken primarily -- LGTM. :)

**Pull request merged**